src/mrt/project.rs

```diff
@@ -0,0 +1,43 @@
+use std::path::PathBuf;
+use glob::glob;
+
+use crate::package::Package;
+use crate::manifest::Manifest;
+
+pub struct Project {
+    path: PathBuf,
+    manifest: Manifest,
+}
+
+impl Project {
+    pub fn new(path: PathBuf) -> Project {
+        Project {
+            path,
+            manifest: Manifest::new()
+        }
+    }
+
+    pub fn get_packages(&self) -> Vec<Package> {
+
+        let mut packages: Vec<Package> = vec![];
+        
+        for package_glob in &self.manifest.packages {
+            let full_glob = self.path.parent().unwrap()
+                .join(&package_glob);
+        
+            for entry in glob(&full_glob.to_str().unwrap())
+                    .expect("Failed to read glob pattern") {
+                match entry {
+                    Ok(path) => {
+                        if path.exists() && path.is_dir() {
+                            packages.push(Package::from_package_path(path));
+                        }
+                    },
+                    Err(e) => println!("{:?}", e),
```

Comment:
Shouldn't you propagate error here, or panic?

Reply:
I think at this point should warn and then ignore. Addressed.